source/js/buyers-guide/template-js-handler/mobile-search-bar.js

```diff
@@ -25,8 +25,8 @@ export default () => {
   ) {
     setupSearchBar(
       "#pni-mobile-bar",
-      "#pni-mobile-container .clear-icon",
-      `#pni-mobile-container`
+      "#pni-mobile-search-container .clear-icon",
+      `#pni-mobile-search-container`
```

Comment:
Can you add some context why the we are using the backticks ``` for this string? It does not seem to do any [string templating](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

Reply:
Ah i've been sticking with backticks for all use cases. I remember that was how our lining rules set to be too. Do we have a different preference now?

Reply:
Intersting. I am ok either way. The linting rule does not seem to be active though, because all the other strings are not using it.

Reply:
Just quickly checked the codebase. I see mixed uses of backticks and quotes everywhere. Looks like I remember it wrong. It seems it's always been fine either way.

Reply:
No worries. I think I could be a good idea for the future to focus on one style and use that consistent through the code base. Would be good to also make linting check for that.